app/metrics.yaml

```diff
@@ -137,6 +137,7 @@ events:
           show_search_shortcuts, open_links_in_a_private_tab (bug in
           implementation https://github.com/mozilla-mobile/fenix/issues/7384),
           pref_key_sync_logins, pref_key_sync_bookmarks, pref_key_sync_history,
+          pref_key_show_voice_search,
```

Comment:
Does this not get updated in the metrics.md? I don't see a change in that file included in this PR.

Reply:
Think I just need to do a rebuild of the project to generate. Let's see what Vesta responds to the other comment :)